osu.Game/Screens/Play/HUD/PlayerAvatar.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Configuration;
+using osu.Game.Localisation.SkinComponents;
+using osu.Game.Skinning;
+using osu.Game.Users.Drawables;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public partial class PlayerAvatar : CompositeDrawable, ISerialisableDrawable
+    {
+        [SettingSource(typeof(SkinnableComponentStrings), nameof(SkinnableComponentStrings.CornerRadius), nameof(SkinnableComponentStrings.CornerRadiusDescription))]
+        public new BindableFloat CornerRadius { get; set; } = new BindableFloat
+        {
+            MinValue = 0,
+            MaxValue = 64,
+            Precision = 0.01f
```

Comment:
I'd argue that the values should be fractional/percentages here. The fact that the avatar has a size of 128 is completely arbitrary, and the avatar can be resized anyways. I'd do a range of [0. 0.5] and multiply that by 128 to get the final size.